command-expansion-server/package.json

```diff
@@ -21,6 +21,7 @@
     "body-parser": "^1.19.2",
     "express": "^4.17.2",
     "graphql": "^16.3.0",
+    "graphql-request": "^4.0.0",
```

Comment:
Is the `graphql` dependency being used? If not then maybe it can be fully replaced with the `graphql-request` dep.?

Reply:
yeah, `graphql` is used to build schemas which we are not doing in this part of the server. @dyst5422 where you using this library somewhere else?

Reply:
Removed